articles/openshift/oshift-how-apply-security-best-practices.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to apply security best practices in OpenShift
+description: Apply security best practices in OpenShift
+services: openshift
+author: Mudasar Hussain
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Apply security best practices in OpenShift
+toc_fullpath: How To/apply-security-best-practices.md
+toc_mdlink: oshift-how-apply-security-best-practices.md
+---
+
+# How to apply security best practices in OpenShift
+
+## Overview
+
+This knowledge centre article sets out some of the best practices which can be adhered to in order to minimise the attack vector to a UKCloud OpenShift cluster and its workloads. As the nature of running container-based workloads reduces the threat to your underlying infrastructure, it is not completely eliminated, as such great care and attention must be taken when designing, developing and deploying your container-based workloads to minimise the risks to your container-based deployments in UKCloud OpenShift. Unfortunately, as is the nature of technology any cloud-based service is vulnerable to exploits from both hackers and insiders just the same as traditional environments. There are risks of data stealing, crypto mining, ransomware and service disruption to name a few.
+
+### Intended audience
+
+This article is at everybody who is using UKCloud OpenShift. It will highlight just some of the best practices to keep OpenShift workloads secure.
```

Comment:
This article is intended for anyone who is using UKCloud for OpenShift. It highlights some of the best practices to keep OpenShift workloads secure.